subcommands/keys/rotate_root.go

```diff
@@ -52,6 +55,11 @@ func doRotateRoot(cmd *cobra.Command, args []string) {
 	root, err := api.TufRootGet(factory)
 	subcommands.DieNotNil(err)
 
+	if doRootSync {
+		subcommands.DieNotNil(syncProdRoot(factory, *root, creds))
+		return
+	}
```

Comment:
As I understand an API doesn't allow to rotate CI root if a prod root is out of sync (that's good).

So, when we handle that 409 status code - I'd like to show a message to a user which shows instructions how to get around that problem.
Smth like `Your production root role is out of sync.  To fix this, please, run "fioctl rotate root --sync-prod"`

Reply:
i'm working on this now. I actually need to consolidate a bit of replicated logic. should have it in the next day as i catch up

Reply:
I've just done a force push to change this some. 
1) consolidate logic for updating root - https://github.com/foundriesio/fioctl/pull/89/commits/fb92c726b046137d8db2dadcbdbe1a82884b63b3
2) the original PR - https://github.com/foundriesio/fioctl/pull/89/commits/c0f32263a693e6b3baed8b66cf855c470eaefd02
3) the fix you asked for here - https://github.com/foundriesio/fioctl/pull/89/commits/67fcd9e70660e722f958dcb8fe4fc02716498561

I can squash 3 in with 2 if you think this looks good.